assets/terraform/aws/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# File passed to VM at creation time
+#
+apt update 
+apt install -y python-pip lvm2
+pip install --upgrade awscli
+
+set -euo pipefail
+
+pvcreate -ff /dev/xvde
+vgcreate data /dev/xvde
+lvcreate -n var_lib_gravity -l 20%VG data
+lvcreate -n var_lib_data -l 20%VG data
+lvcreate -n etcd -l 60%VG data
```

Comment:
In practice this needs a real 'physical' device of it's own. Instance storage if the instance has it, or a 50GB 2500IOPS io1 SSD on AWS.